src/RDF/RdfNamespaces.php

```diff
@@ -10,6 +10,12 @@ final class RdfNamespaces
 {
     public static function register(): void
     {
+        // Delete the initial 'dc' prefix for dcterms, so the other initial 'dcterms' namespace is used instead.
+        RdfNamespace::delete('dc');
+
+        // Set custom namespaces.
+        RdfNamespace::set('adms', 'http://www.w3.org/ns/adms#');
         RdfNamespace::set('locn', 'http://www.w3.org/ns/locn#');
+        RdfNamespace::set('generiek', 'https://data.vlaanderen.be/ns/generiek/#');
```

Comment:
So the namepace `dcterms` is already known without setting it right?

Reply:
Yep, it's one of the defaults provided by EasyRDF itself in the `EasyRdf\RdfNamespace` class in its `$initial_namespaces` property. It also sets the `dc` namespace by default, which I had to remove because they were both the same and it would otherwise always use `dc` instead of `dcterms` (which we usually use in examples of OSLO data etc so that's probably clearer than `dc`).